constants/additionalChainRegistry/chainid-41295.js

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "rootVX testnet",
+  "chain": "rootVX",
+  "rpc": [
+    "http://34.60.253.118:9545",
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://rootvx.com",
+  "shortName": "rootVX",
```

Comment:
Adding a T for the testnet (shortname is rootVXT)
Once mainnet is live you can use rootVX as shortName